_notebooks/2022-02-21-Exploring-Torch.lerp.ipynb

```diff
@@ -0,0 +1,387 @@
+    }
+   },
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here is an example in the [mixup paper](https://arxiv.org/pdf/1710.09412.pdf) of lerp being used in practice: ![image.png](attachment:image.png)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 52,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2022-02-19T18:34:36.791060Z",
+     "start_time": "2022-02-19T18:34:36.747773Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "x_i = torch.randn((3,512,512)) #simulated image #1\n",
+    "x_j = torch.randn((3,512,512)) #simulated image #2\n",
```

Comment:
why not make a dog pic and cat pic, and make a dog cat? it'll be helpful for visual learners out there.

Reply:
Added a picture of April, if you want to add Rex as well, just give me a picture and I'll get him added otherwise it will be april vs noise which I think makes sense too.